examples/hello-world-helm/service_account_runner.yaml

```diff
@@ -15,3 +16,17 @@ subjects:
 - kind: ServiceAccount
   name: eunomia-runner
   namespace: eunomia-hello-world-demo
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: eunomia-demo-runner-cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: eunomia-runner
+  namespace: eunomia-hello-world-demo
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
+---
```

Comment:
What's the reason for adding this? can you please try to explain in the commit message? I really don't have a slightest idea :( things seemed to be working before, it would really help me understand this code if I knew what's the intention behind it (a.k.a. "why?")